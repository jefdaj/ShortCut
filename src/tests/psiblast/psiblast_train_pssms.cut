pcc6803 = load_faa "data/6803genes.faa"
pcc7942 = load_faa "data/7942genes.faa"
queries = split_faa (load_faa "data/7942genes10.faa")

# TODO write this:
pssms   = psiblast_train_pssms 0.1 queries [pcc6803, pcc7942]

result  = pssms
