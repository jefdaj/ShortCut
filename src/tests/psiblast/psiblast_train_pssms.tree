.
|-- cache
|   |-- lines
|   |   |-- 04b6898a46.txt
|   |   |-- 5649fe82c8.txt
|   |   |-- 5758625753.txt
|   |   |-- 74a09cfaee.txt
|   |   |-- 7b5fc2e36a.txt
|   |   |-- 876a82283a.txt
|   |   |-- 8a011291e9.txt
|   |   |-- 8c65b6424c.txt
|   |   |-- b2cdb3795b.txt
|   |   |-- bdc783547b.txt
|   |   |-- c0830109d9.txt
|   |   |-- c711209c54.txt
|   |   |-- e00fb07ddb.txt
|   |   |-- e9819ea256.txt
|   |   |-- ed55d6e9a8.txt
|   |   |-- ef87d3e725.txt
|   |   `-- f0af818a51.txt
|   |-- load
|   |   |-- 32841598b2.faa -> $WORKDIR/data/7942genes10.faa
|   |   `-- a810086108.faa -> $WORKDIR/data/7942genes.faa
|   |-- makeblastdb
|   |   `-- e00fb07ddb
|   |       |-- e00fb07ddb.pdb.list.phr
|   |       |-- e00fb07ddb.pdb.list.pin
|   |       `-- e00fb07ddb.pdb.list.psq
|   |-- map
|   |   |-- 1f687fd5c7.pssm -> ../../cache/lines/ed55d6e9a8.txt
|   |   |-- 3fa06f06cc.pssm -> ../../cache/lines/876a82283a.txt
|   |   |-- 4cd1a54f7d.pssm -> ../../cache/lines/7b5fc2e36a.txt
|   |   |-- 7066cdce18.pssm -> ../../cache/lines/c711209c54.txt
|   |   |-- 8b344b93f1.pssm -> ../../cache/lines/5649fe82c8.txt
|   |   |-- 8d2e82b0ad.pssm -> ../../cache/lines/b2cdb3795b.txt
|   |   |-- a08f7b4b10.pssm -> ../../cache/lines/8c65b6424c.txt
|   |   |-- a331168f51.pssm -> ../../cache/lines/c0830109d9.txt
|   |   |-- c13f4f52b6.pssm -> ../../cache/lines/5758625753.txt
|   |   `-- ee6a68c416.pssm -> ../../cache/lines/04b6898a46.txt
|   `-- split_faa
|-- exprs
|   |-- list
|   |   `-- fa4677789c.faa.list -> ../../cache/lines/e00fb07ddb.txt
|   |-- load_faa
|   |   |-- 18a54fafca4.faa
|   |   |-- 2d88028e40.faa -> ../../cache/load/32841598b2.faa
|   |   |-- 387f499ddc.faa -> ../../cache/load/a810086108.faa
|   |   |-- 55895cbd9ee.faa
|   |   |-- 5b316091639.faa
|   |   |-- 6f29186ec90.faa
|   |   |-- 82645ed6f61.faa
|   |   |-- 9500e84b092.faa
|   |   |-- a3a42267174.faa
|   |   |-- b2a261c0c1f.faa
|   |   |-- cab744a6f9d.faa
|   |   `-- e225e53d82b.faa
|   |-- makeblastdb_prot_all
|   |   `-- ced061f0b2.pdb.list -> ../../cache/lines/74a09cfaee.txt
|   |-- num
|   |   `-- efc8362dcc.num -> ../../cache/lines/e9819ea256.txt
|   |-- psiblast_train_pssms
|   |   `-- 4f7795b38a_a726be61de_3cbf6d2446.pssm.list -> ../../cache/lines/8a011291e9.txt
|   |-- split_faa
|   |   `-- 1af4d085e7.faa.list -> ../../cache/lines/f0af818a51.txt
|   `-- str
|       |-- 1d8a61c770.str -> ../../cache/lines/bdc783547b.txt
|       `-- db90f32127.str -> ../../cache/lines/ef87d3e725.txt
`-- vars
    |-- pcc7942.faa -> ../exprs/load_faa/387f499ddc.faa
    |-- pssms.pssm.list -> ../exprs/psiblast_train_pssms/4f7795b38a_a726be61de_3cbf6d2446.pssm.list
    |-- queries.faa.list -> ../exprs/split_faa/1af4d085e7.faa.list
    `-- result -> ../vars/pssms.pssm.list

16 directories, 55 files
