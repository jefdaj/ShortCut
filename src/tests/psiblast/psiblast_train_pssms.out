[>Synpcc7942_0001 DNA polymerase III subunit beta (trained on e00fb07ddb.pdb.l...
Last position-specific scoring matrix computed, weighted observed percentages...
            A   R   N   D   C   Q   E   G   H   I   L   K   M   F   P   S   T...
    1 M    -1  -1  -2  -3  -1   0  -2  -3  -2   1   2  -1   5   0  -2  -1  -1...
    2 L    -1  -2  -3  -4  -1  -2  -3  -4  -3   2   4  -2   2   0  -3  -2  -1...
...
,
 >Synpcc7942_0002 hypothetical protein (trained on e00fb07ddb.pdb.list)
Last position-specific scoring matrix computed, weighted observed percentages...
            A   R   N   D   C   Q   E   G   H   I   L   K   M   F   P   S   T...
    1 M    -1  -1  -2  -3  -1   0  -2  -2  -1   1   2  -1   5   0  -2  -1  -1...
    2 A     4  -1  -1  -2   0  -1  -1   0  -2  -1  -1  -1  -1  -2  -1   1   0...
...
,
 >Synpcc7942_0003 phosphoribosylformylglycinamidine synthase II (trained on e0...
Last position-specific scoring matrix computed, weighted observed percentages...
            A   R   N   D   C   Q   E   G   H   I   L   K   M   F   P   S   T...
    1 M    -1  -1  -2  -3  -1   0  -2  -3  -2   1   2  -1   5   0  -2  -1  -1...
    2 T     0  -1   0  -1  -1  -1  -1  -2  -2  -1  -1  -1  -1  -2  -1   1   5...
...
,
 >Synpcc7942_0004 amidophosphoribosyltransferase (trained on e00fb07ddb.pdb.list)
Last position-specific scoring matrix computed, weighted observed percentages...
            A   R   N   D   C   Q   E   G   H   I   L   K   M   F   P   S   T...
    1 M    -1  -1  -2  -3  -1   0  -2  -3  -2   1   2  -1   5   0  -2  -1  -1...
    2 I    -1  -3  -3  -3  -1  -3  -3  -4  -3   4   2  -3   1   0  -3  -2  -1...
...
,
 >Synpcc7942_0005 hypothetical protein (trained on e00fb07ddb.pdb.list)
Last position-specific scoring matrix computed, weighted observed percentages...
            A   R   N   D   C   Q   E   G   H   I   L   K   M   F   P   S   T...
    1 M    -1  -2  -2  -3  -2   0  -2  -3  -2   1   2  -2   6   0  -3  -2  -1...
    2 L    -2  -2  -4  -4  -1  -2  -3  -4  -3   2   4  -3   2   0  -3  -3  -1...
...
,
 >Synpcc7942_0006 hypothetical protein (trained on e00fb07ddb.pdb.list)
Last position-specific scoring matrix computed, weighted observed percentages...
            A   R   N   D   C   Q   E   G   H   I   L   K   M   F   P   S   T...
    1 M    -1  -1  -2  -3  -1   0  -2  -3  -2   1   2  -1   5   0  -2  -1  -1...
    2 T     0  -1   0  -1  -1  -1  -1  -2  -2  -1  -1  -1  -1  -2  -1   1   5...
...
,
 >Synpcc7942_0007 hypothetical protein (trained on e00fb07ddb.pdb.list)
Last position-specific scoring matrix computed, weighted observed percentages...
            A   R   N   D   C   Q   E   G   H   I   L   K   M   F   P   S   T...
    1 M    -1  -1  -2  -3  -1   0  -2  -3  -2   1   2  -1   5   0  -2  -1  -1...
    2 T     0  -1   0  -1  -1  -1  -1  -2  -2  -1  -1  -1  -1  -2  -1   1   5...
...
,
 >Synpcc7942_0008 hypothetical protein (trained on e00fb07ddb.pdb.list)
Last position-specific scoring matrix computed, weighted observed percentages...
            A   R   N   D   C   Q   E   G   H   I   L   K   M   F   P   S   T...
    1 M    -1  -1  -2  -3  -1   0  -2  -3  -2   1   2  -1   6   0  -3  -2  -1...
    2 L    -2  -2  -4  -4  -1  -2  -3  -4  -3   2   4  -3   2   0  -3  -3  -1...
...
,
 >Synpcc7942_0009 argininosuccinate synthase (trained on e00fb07ddb.pdb.list)
Last position-specific scoring matrix computed, weighted observed percentages...
            A   R   N   D   C   Q   E   G   H   I   L   K   M   F   P   S   T...
    1 M    -1  -1  -2  -3  -1   0  -2  -3  -2   1   2  -1   5   0  -2  -1  -1...
    2 G     0  -2   0  -1  -2  -2  -2   6  -2  -4  -4  -2  -3  -3  -2   0  -2...
...
,
 >Synpcc7942_0010 hypothetical protein (trained on e00fb07ddb.pdb.list)
Last position-specific scoring matrix computed, weighted observed percentages...
            A   R   N   D   C   Q   E   G   H   I   L   K   M   F   P   S   T...
    1 M    -1  -1  -2  -3  -1   0  -2  -3  -2   1   2  -1   6   0  -3  -2  -1...
    2 S     1  -1   1   0  -1   0   0   0  -1  -2  -3   0  -2  -2  -1   4   1...
...
]
